examples/build.sh

```diff
@@ -36,25 +49,25 @@ cargo_build() {
 }
 
 # wasm-pack build
-if [[ $EXAMPLE == *_wp ]]; then
+if [[ $EXAMPLE == *_wp ]] || [[ $EXAMPLE == todomvc ]]; then
```

Comment:
I'm rather reluctant to do that because of all the links refer to the example directory would lead to 404. But such decision is to be taken by the core teammates.

Reply:
I've noticed that the `todomvc` example can be built with `wasm-bindgen` without a problem. I don't think we even need to make this change.